app/schema/answer.py

```diff
@@ -100,3 +100,14 @@ def mandatory_error(self, state):
         state.errors = []
         state.errors.append(self.questionnaire.get_error_message('MANDATORY', self.id))
         state.is_valid = False
+
+    @staticmethod
+    def _valid_option_selected(state_input, options):
```

Comment:
Decided not to try to predict the future and put it in the only place it is currently used checkbox_answer.py